src/libraries/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -151,6 +152,7 @@ internal static class KnownColorTable
             0xFFF5F5F5,     // WhiteSmoke
             0xFFFFFF00,     // Yellow
             0xFF9ACD32,     // YellowGreen
+            0xFF663399,     // RebeccaPurple
```

Comment:
You forgot to remove the entry above that you added before, so this is now duplicate.